src/d3d12/d3d12_heap.cpp

```diff
@@ -11,22 +11,22 @@ namespace wr::d3d12
 		return frame % (8 * 8);
 	}
 
-	static void SetFrame(std::vector<uint64_t>* frames, std::uint64_t frame) {
+	static void SetFrame(std::vector<uint64_t>* bitmap, std::uint64_t frame) {
```

Comment:
These function should preferable be in the `internal` namespace. This makes doxygen documentation more readable.